wiki/People/Project_Loved_Team/en.md

```diff
@@ -26,6 +26,13 @@ Some team additions may be added manually on a case-by-case basis in order to fi
 
 The criteria for joining the team vastly differs from role to role, but proficiency and community presence is always sought after.
 
+## Benefits
+
+Project Loved Team members are granted the following benefits:
```

Comment:
"Project Loved Team members are given profile badges that display the number of years [...]"
no need for 1-item list